build.gradle

```diff
@@ -102,5 +102,5 @@ compileJava {
 }
 
 test {
-  useJUnitPlatform()
+  forkEvery = 1
```

Comment:
I am skeptical about this? What is the need for this?

Reply:
it is redundant, you may delete this line.